python/pyspark/sql/functions.py

```diff
@@ -215,7 +215,7 @@ def _():
                    ' the expression in a group.',
     'stddev_pop': 'Aggregate function: returns population standard deviation of' +
                   ' the expression in a group.',
-    'variance': 'Aggregate function: returns the population variance of the values in a group.',
+    'variance': 'Aggregate function: returns the unbiased variance of the values in a group.',
     'var_samp': 'Aggregate function: returns the unbiased variance of the values in a group.',
```

Comment:
Shall we add `unbiased sample variance` like the following two sentences?
```
'variance': 'Aggregate function: returns the unbiased sample variance of the values in a group.',
'var_samp': 'Aggregate function: returns the unbiased sample variance of the values in a group.',
```

Reply:
Hello! It seems that sample variance is an unbiased estimator of the variance, so unbiased sample variance might be an unnecessary repetition?

Reply:
The terminology `unbiased sample variance` is more precise. Please see https://en.wikipedia.org/wiki/Variance .

Reply:
Just say that 'variance' is an alias for 'var_samp' here rather than duplicate anything.

- `var_pop` in databases is the variance of the values (e.g. divided by n). It's 'correct' if the data it's given is the whole 'population', as it is simply the value of the statistic.
- `var_samp` treats the data as a sample of a larger unknown population. It's still returning the _population_ variance, but an unbiased estimate of it (e.g. divided by n-1).

Therefore the `var_samp` description is maybe best as "an unbiased estimate of the population variance given the values as a sample". "Unbiased sample variance" is OK too though I've never liked that description. The sample variance is just a statistic of given values and can't be 'biased'. It's only biased or not as an estimator of population variance.

Anyway, I'm OK fixing all the docs accordingly, at least to read "unbiased sample variance".

Reply:
+1 for @srowen 's recommendation about `alias`.